encord/orm/base_dto/base_dto_pydantic_v1.py

```diff
@@ -15,7 +16,10 @@ class Config:
 
     @classmethod
     def from_dict(cls: Type[T], d: Dict[str, Any]) -> T:
-        return cls.parse_obj(d)  # type: ignore[attr-defined]
+        try:
+            return cls.parse_obj(d)  # type: ignore[attr-defined]
+        except ValidationError as e:
+            raise EncordException(message=str(e)) from e
```

Comment:
I'm not sure if we're throwing this anywhere so far (I haven't checked). In any case seems to me like we might want to throw a more specifc error